app/exceptions/atmosphere/unsupported_exception.rb

```diff
@@ -0,0 +1,3 @@
+module Atmosphere
+  class UnsupportedException < Exception; end
+end
```

Comment:
What this change do? It looks like the removed file. Did you only change the name from `app/exceptions/atmosphere/unsuported_exception.rb` to `app/exceptions/atmosphere/unsupported_exception.rb` (with double `pp`)?